.editorconfig

```
@@ -0,0 +1,27 @@
+# This file is for unifying the coding style for different editors and IDEs
+# editorconfig.org
+
+# WordPress Coding Standards
+# http://make.wordpress.org/core/handbook/coding-standards/
+
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+indent_style = tab
```

```suggestion
indent_style = tab
indent_size = 4
```

GitHub displays tabs based on this info 👁️

---

I can amend/remove that if it needed.

---

It would be nice. Here GitHub displays TAB characters are 8 wide!